WebAppServiceBus/WebAppServiceBus/Controllers/HomeController.cs

```diff
@@ -59,29 +57,34 @@ public async Task<ActionResult> Send(ServiceBusMessageData messageInfo)
                 return RedirectToAction("Index");
             }
 
-            var tokenProvider = TokenProvider.CreateManagedServiceIdentityTokenProvider();
-            QueueClient sendClient = new QueueClient($"sb://{Config.Namespace}.servicebus.windows.net/", Config.Queue, tokenProvider);
-            await sendClient.SendAsync(new Message(Encoding.UTF8.GetBytes(messageInfo.MessageToSend)));
-            await sendClient.CloseAsync();
+            ServiceBusClient client = new ServiceBusClient(Config.Namespace, new DefaultAzureCredential());
+
+            ServiceBusSender sender = client.CreateSender(Config.Queue);
+            ServiceBusMessage[] messages = new ServiceBusMessage[] {
+                new ServiceBusMessage(messageInfo.MessageToSend)
+            };
+            await sender.SendMessagesAsync(messages);
+            await sender.CloseAsync();
 
+            await ReceivedMessageStore.InitializeAsync(Config);
```

Comment:
This should only happen once so we should move it into the constructor and store off the task that can be awaited from the Send method:
_initializeTask = ReceivedMessageStore.InitializeAsync(config);

In Send:
await _initializeTask.ConfigureAwait(false);